### PR TITLE
Transition from mailgun org to gubernator-io org

### PR DESCRIPTION
**Pull request by thrawn01**
### Purpose
This is to transition the repo from github.com/mailgun/gubernator to github.com/gubernator-io/gubernator such that adding and removing contributors is simplified and does not require maintainers have access to the mailgun org.

### Implementation
* Transition public container from `ghcr.io/mailgun/gubernator` to `ghcr.io/gubernator-io/gubernator`
* Rename all import paths from `github.com/mailgun/gubernator` to `github.com/gubernator-io/gubernator`
*Ensure links to all external and github resources point to `https://raw.githubusercontent.com/gubernator-io/gubernator/master`
* Added `CONTRIBUTING.md` and `CODE_OF_CONDUCT.md`

### Manual Process
- [x] Ensure existing tags remain
- [x] Transition old container tags to new repo
- [x] Ensure github actions work as expected
- [ ] Create a new release in `github.com/gubernator-io/gubernator` which refers to the old repo release log
- [ ] Validate docker containers for the new release are built and can be accessed via `ghcr.io/gubernator-io/gubernator`
- [ ] Re-enable `check-version.sh` script in github actions after new release.

### CODE REVIEW

File: CODE_OF_CONDUCT.md [hunk 1]
```diff
@@ -0,0 +1,135 @@
...
+an individual is officially representing the community in public spaces.
+Examples of representing our community include using an official email address,
+posting via an official social media account, or acting as an appointed
+representative at an online or offline event.
+
+## Enforcement
+
+Instances of abusive, harassing, or otherwise unacceptable behavior may be
+reported to the community leaders responsible for enforcement at
+[INSERT CONTACT METHOD].
```
Comment (Baliedge) [inline CODE_OF_CONDUCT.md:61]: `[INSERT CONTACT METHOD]`

Comment (Baliedge) [reply CODE_OF_CONDUCT.md:61]: Who are the "community leaders" and how I do get to be one of these awesome people?

File: go.mod [hunk 2]
```diff
@@ -1,4 +1,4 @@
-module github.com/mailgun/gubernator/v2
+module github.com/gubernator-io/gubernator/v2
```
Comment (Baliedge) [inline go.mod:1]: Can we do away with `/v2`?

**Review by Baliedge — CHANGES_REQUESTED** [review]

File: CODE_OF_CONDUCT.md (hunk 1 above)
Comment (thrawn01) [reply CODE_OF_CONDUCT.md:61]: You are a repo maintainer in GitHub and you are in the code owner file.

File: go.mod (hunk 2 above)
Comment (thrawn01) [reply go.mod:1]: I like the idea, but then I would need to clear all the git tags and containers versions. It might be confusing.

Comment (Baliedge) [reply go.mod:1]: Then how about `/v3`?

File: CODE_OF_CONDUCT.md (hunk 1 above)
Comment (Baliedge) [reply CODE_OF_CONDUCT.md]: Oh, so I'm already awesome.  Got it. 👍🏻

File: go.mod (hunk 2 above)
Comment (thrawn01) [reply go.mod:1]: I'll kick it to v3 when the DUH-RPC stuff drops. 😋

**Review by Baliedge — APPROVED** [review]

**Comment by thrawn01** [comment]
I'm holding this PR until we merge and update the other pending PR's on the old repo.

File: benchmark_cache_test.go [hunk 3]
```diff
@@ -6,7 +6,7 @@ import (
 	"testing"
 	"time"
 
-	gubernator "github.com/mailgun/gubernator/v2"
+	"github.com/gubernator-io/gubernator/v2"
```
Comment (Baliedge) [inline benchmark_cache_test.go:9]: Recent versions of golangci-lint complain about this and prefer the explicit import name.

**Pull request merged**